updater/spec/dependabot/integration_spec.rb

```diff
@@ -398,8 +398,8 @@
         expect(log_message).to include(
           "created",
           "dummy-git-dependency",
-          "from 20151f9b67c8a04461fa0ee28385b6187b86587b",
-          "to c0e25c2eb332122873f73acb3b61fb2e261cfd8f"
+          "from v1.0.0",
```

Comment:
Could you explain why this is populating versions now instead of shas? That doesn't match the example in the PR so I'm not following this part.

Reply:
Oh, sorry, I only found about this after creating the PR when I saw this spec failure in the updater specs.

This is just another nice side effect of using a "humanized version" instead of the "raw version". In the case of `git` dependencies, the new logic will checker whether there's a reference available instead in the raw commit sha and use that if available (and has been changed). It's this logic here: https://github.com/dependabot/dependabot-core/pull/6996/files#r1157787124.

Reply:
As an example, if a Gemfile is being changed from 

```ruby
gem "foo", gitHub: "user/repo", tag: "v1.0.0"
```

to

```ruby
gem "foo", gitHub: "user/repo", tag: "v1.1.0"
```

we'll now use those in the summary instead of the locked commit SHAs.